core/src/main/java/apoc/coll/Coll.java

```diff
@@ -69,6 +70,9 @@ public class Coll {
     @Context
     public Transaction tx;
 
+    @Context
+    public ProcedureCallContext procedureCallContext;
```

Comment:
I forgot, do `ProcedureCallContext` require allow unrestricted access?

Reply:
Yeah, it is marked as safe or whatever the terminology is :)